ecosystem/typescript/sdk/src/aptos_types/transaction.ts

```diff
@@ -208,6 +208,39 @@ export class EntryFunction {
   }
 }
 
+export class MultiSig {
+  /**
+   * Contains the payload to run a multisig account transaction.
+   * @param multisig_address The multisig account address the transaction will be executed as.
+   * @param transaction_payload The payload of the multisig transaction. This is optional when executing a multisig transaction
+   *  whose payload is already stored on chain.
+   */
+  constructor(
+    public readonly multisig_address: AccountAddress,
+    public readonly transaction_payload?: EntryFunction | undefined,
```

Comment:
nit: when you have the "?", `transaction_payload?: EntryFunction` is the same as `transaction_payload?: EntryFunction | undefined`. Therefore, the `| undefined` is unnecessary.

Here you can either use `transaction_payload: EntryFunction | undefined` Or `transaction_payload?: EntryFunction`

Reply:
Updated